core/src/main/java/org/apache/spark/util/collection/unsafe/sort/UnsafeSorterSpillWriter.java

```diff
@@ -62,6 +62,8 @@ public UnsafeSorterSpillWriter(
       int fileBufferSize,
       ShuffleWriteMetrics writeMetrics,
       int numRecordsToWrite) throws IOException {
+    // space used by prefix + len is (4 + 8) bytes
```

Comment:
Can we refine this comment to explain more than 12 bytes are required?
For example, `space used by prefix + len + recordLength is more than 4 + 8 bytes`?